comparators/modifiedMonument.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
@srividyacb - you seem to have added both comparators/modifiedMonument.js and comparators/modified-monument.js - can you remove `comparaots/modifiedMonument.js` ?